src/streaming-services/vrtnu-be/VrtnuBeParser.ts

```diff
@@ -0,0 +1,77 @@
+import { Item } from '../../models/Item';
+import { registerScrobbleParser } from '../common/common';
+import { ScrobbleParser } from '../common/ScrobbleController';
+import { VrtnuBeApi } from './VrtnuBeApi';
+
+class _VrtnuBeParser implements ScrobbleParser {
+	id: string;
+	videoId: string;
+	progress: number;
+	isPaused: boolean;
+
+	constructor() {
+		this.id = '';
+		this.videoId = '';
+		this.progress = 0.0;
+		this.isPaused = false;
+	}
+
+	parseProgress = (): number => {
+		let progress = 0.0;
+		const scrubber: HTMLElement | null = document.querySelector('.vjs-play-progress');
+
+		if (scrubber) {
+			progress = parseFloat(scrubber?.style.left);
+			this.progress = progress;
+		}
+
+		return progress;
+	};
+
+	parseItem = (): Item | undefined => {
+		const serviceId = 'vrtnu-be';
+		let showTitle: string | null = null;
+		let seasonOrYear: string | null = null;
+		let subTitle: string | undefined = undefined;
+		const matches = /\/a-z\/(.+)\/(.+)\/(.+)\//.exec(location.href);
+
+		if (matches) {
+			[, showTitle, seasonOrYear, subTitle] = matches;
+		}
+
+		const id = subTitle ?? '';
+		const title = showTitle?.split('-').join(' ') ?? '';
+		const episodeTitle = '';
+		const isSeason = subTitle?.lastIndexOf('s' + String(seasonOrYear) + 'a') ?? 0;
```

Comment:
It's not good to do conversions like this (`String(...)`). If you're trying to account for when `seasonOrYear` is null, isn't it better to check for that explicitly? You could just do `(seasonOrYear ?? '...')`.

Also, it's confusing that the variable is named `isSeason` but holds a number instead of a boolean. It's probably better to rename it to `seasonIndex` or something.

Reply:
The problem with this was that I got an error on " + seasonOrYear + "

`Operands of '+' operation must either be both strings or both numbers. Consider using a template literal  @typescript-eslint/restrict-plus-operands`

This seemed to be the quick fix for that. If you have a proposition to change it a better way, feel free :)

SeasonOrYear is never null by the way. There's always a value on the page / in the url.

seasonIndex is indeed a better name for this!

Reply:
But you define it as `let seasonOrYear: string | null = null;`, so even if, in practice, it will always eventually become a string, TypeScript doesn't know that. The solution should be what I suggested above: `(seasonOrYear ?? '')`

Reply:
Oh oops, I skipped the last part about "(seasonOrYear ?? '')"
I'll change it and also change the naming